docs/develop-resources/scenarios/new-field.md

```diff
@@ -11,23 +11,22 @@ REPO_ROOT="$(git rev-parse --show-toplevel)"
 cd $REPO_ROOT/dev/tools/controllerbuilder
 
 go run . update-types \
-    --parent-message-full-name "google.monitoring.dashboard.v1.Dashboard" \
-    --new-field "row_layout" \
+    --parent-message "google.monitoring.dashboard.v1.Dashboard" \
+    --field-to-insert "row_layout" \
     --api-dir ${REPO_ROOT}/apis/monitoring/v1beta1
 ```
 
-* `--parent-message-full-name`
+* `--parent-message`
```

Comment:
Suggest using no more than 2 words for flag name. Otherwise it would be very hard to use.
how about `parent` and adding an example in the desription.

Reply:
Yeah we can use `parent`.